pages/src/common/badge.less

```
@@ -19,6 +20,12 @@
     display: none;
   }
 
+  &-sidebar {
+    position: absolute;
+    top: 38%;
+    left: 24px;
```

```suggestion
    top: 50%;
    left: 24px;
    transform: translateY(-50%);
```
This way, it will not sends on badge size